core/player/src/controllers/error/controller.ts

```diff
@@ -0,0 +1,159 @@
+import { SyncBailHook } from "tapable-ts";
+import type { Logger } from "../../logger";
+import type { DataController } from "../data/controller";
+import type { PlayerError, ErrorMetadata, ErrorSeverity } from "./types";
+
+export interface ErrorControllerHooks {
+  /**
+   * Fired when any error is captured
+   * - Called in order for each tapped plugin
+   * - Return true to bail and prevent error state navigation
+   * - Return undefined/false to continue to next handler
+   * - Once true is returned, no further plugins are called
+   */
+  onError: SyncBailHook<[PlayerError], boolean | undefined>;
+}
+
+export interface ErrorControllerOptions {
+  /** Optional logger for error operations */
+  logger?: Logger;
+  /** Required for setting data.errorState */
```

Comment:
Do we need to add middleware to prevent writing to that namespace from the Flow?